### PR TITLE
feat(ios): add ios platform

### LINKED ISSUE #5204: Native Mobile - configure iOS project

### PR DESCRIPTION
**Pull request by jasonwoodland**
### What this PR does 📖
- Add capacitor generated iOS project
  - Set light statusbar
  - Set dark keyboard
  - Animate `#app` `paddingBottom` to compensate for software keyboard change event
  - Compensate for iOS safe area view margins ***coming soon***

### Which issue(s) this PR fixes 🔨
- Resolve #5204 

### Special notes for reviewers 🗒️
Test by running:
```sh
yarn install
yarn generate
npx cap open ios
```

### Additional comments 🎤
-

### CODE REVIEW

**Comment by josephmcg** [comment]
this is tricky to get running, may need to wait until next week.
if anyone else wants to screw around with it,you need cocoapods

**Comment by josephmcg** [comment]
I had to make a couple changes, Jason. definePageMeta isn't supported in bridge yet, so i downgraded to options api so we can still use the safearea middleware.

I also noticed the safearea was funky on regular mobile browser, so i added a css max check. For some reason it was ending up as 0 if there was no safe area var

File: pages/mobile/chat/_id.vue [hunk 1]
```diff
@@ -125,6 +125,7 @@ export default Vue.extend({
   display: flex;
   flex: 1;
   overflow: hidden;
+  padding-top: var(--safe-area-inset-top);
```
Comment (josephmcg) [inline pages/mobile/chat/_id.vue:128]: could we apply these at layout level instead? rather than every page

File: .prettierignore [hunk 2]
```diff
@@ -9,4 +9,5 @@ plugins
 .nuxt
 pnpm-lock.yaml
 android/
+ios/
```
Comment (stavares843) [inline .prettierignore:12]: thanks for adding 🔨

File: ios/App/App/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
+{
+  "images" : [
+    {
+      "size" : "20x20",
+      "idiom" : "iphone",
+      "filename" : "AppIcon-20x20@2x.png",
```
Comment (stavares843) [inline ios/App/App/Assets.xcassets/AppIcon.appiconset/Contents.json:6]: we need to update the icons also 🎨

**Comment by WanderingHogan** [comment]
the biggest things to get this running are:
have xcode with licenses accepted
install cocopods through homebrew

i added ios-run to package.json to help, too

**Comment by stavares843** [comment]
we could add a bash script with all the pods stuff as well

**Comment by jasonwoodland** [comment]
You have to set the team to personal in signing for now:
Screen Shot 2022-10-15 at 10 45 17 am

**Comment by stavares843** [comment]
🔨

**Pull request merged**